packages/ui/src/common/components/BlockTime/BlockInfo.tsx

```diff
@@ -13,10 +13,23 @@ export interface BlockInfoProp {
   block: Block
   lessInfo?: boolean
   inline?: boolean
+  noLink?: boolean
 }
 
-export const BlockInfo = ({ block, lessInfo, inline }: BlockInfoProp) => {
+export const BlockInfo = ({ block, lessInfo, inline, noLink }: BlockInfoProp) => {
   const [endpoints] = useNetworkEndpoints()
+
+  // TODO UN-DRY
+  if (noLink)
+    return (
+      <BlockInfoContainer lessInfo={lessInfo} inline={inline}>
+        <BlockIcon />
+        <span>
+          {lessInfo && 'Block'} {formatTokenValue(block.number)}
+        </span>
+      </BlockInfoContainer>
+    )
```

Comment:
review hint: the reasoning was that BlockLink produced a link as child of the ElectionRow link. However i don't like that solution and will move the block explorer link to a tooltip instead.